js/core/src/Blocks.svelte

```diff
@@ -720,19 +720,19 @@
 
 <div class="wrap" style:min-height={app_mode ? "100%" : "auto"}>
 	<div class="contain" style:flex-grow={app_mode ? "1" : "auto"}>
-		<!-- {#if $_layout} -->
-		<MountComponents
-			rootNode={$_layout}
-			{root}
-			{target}
-			{theme_mode}
-			on:mount={handle_mount}
-			{version}
-			{autoscroll}
-			{max_file_size}
-			client={app}
-		/>
-		<!-- {/if} -->
+		{#if $_layout && app.config}
```

Comment:
This is the fix @pngwn but not sure why it was removed from #9339 so can you review?

Reply:
It breaks SSR iirc.

Reply:
@pngwn It doesn't break SSR mode from what I checked. And I also am pushing up some tweaks to make reload mode work on SSR.

### SSR Mode
![ssr_reload_mode](https://github.com/user-attachments/assets/f5396383-7a44-42f2-9b75-91c0f714c8d5)



### NO SSR MODE
![no_ssr_reload_mode](https://github.com/user-attachments/assets/921d8544-65e8-42ca-8679-f2228a9e4e97)